ipn/localapi/localapi.go

```diff
@@ -518,6 +520,28 @@ func (h *Handler) serveGoroutines(w http.ResponseWriter, r *http.Request) {
 	w.Write(buf)
 }
 
+// serveNetmap provides a JSON dump of the last known netmap.
+func (h *Handler) serveNetmap(w http.ResponseWriter, r *http.Request) {
+	// Require write access for privacy reasons.
+	if !h.PermitWrite {
+		http.Error(w, "netmap dump access denied", http.StatusForbidden)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+
+	if nm := h.b.NetMap(); nm != nil {
+		j, err := json.MarshalIndent(nm, "", "\t")
+		if err != nil {
+			http.Error(w, fmt.Sprintf("failed to marshal netmap: %v", err), http.StatusInternalServerError)
+			return
+		}
+		w.Write(j)
+	} else {
+		http.Error(w, "no netmap", http.StatusNotFound)
+	}
+}
+
```

Comment:
Now that we're using the IPN bus, do we still need this?

Reply:
Removed.